### PR TITLE
Correct LastPR and add ssh route

### PR DESCRIPTION
**Pull request by g-boy05**
I correct my PR now ,all is OK now
the problem come from the extra brackets and i add the route for ssh page

### CODE REVIEW

File: src/main/webapp/app/applications/applications.module.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ import {
 @NgModule({
     imports: [
         JhipsterRegistrySharedModule,
-        RouterModule.forRoot([ applicationsState ], {useHash: true})
+        RouterModule.forRoot( applicationsState , {useHash: true})
```
Comment (deepu105) [inline src/main/webapp/app/applications/applications.module.ts]: unwanted space

File: src/main/webapp/app/ssh/ssh.module.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ import {
 @NgModule({
     imports: [
         JhipsterRegistrySharedModule,
-        RouterModule.forRoot([ SSH_ROUTE ], {useHash: true})
+        RouterModule.forRoot( sshState , {useHash: true})
```
Comment (deepu105) [inline src/main/webapp/app/ssh/ssh.module.ts]: unwanted space

**Comment by g-boy05** [comment]
I fix the conflits in this PR #116

**Comment by deepu105** [comment]
I dont see any updates for what I commented. did you forgot to push

**Pull request closed without merging**